lib/node/fullnode.js

```diff
@@ -105,7 +105,8 @@ class FullNode extends Node {
       host: this.config.str('host'),
       port: this.config.uint('port'),
       listen: this.config.bool('listen'),
-      memory: this.config.bool('memory')
+      memory: this.config.bool('memory'),
+      discover: this.config.bool('discover', this.config.bool('listen'))
```

Comment:
I think the style preference for these is to put `this.listen = this.config.bool('listen', false)` at the top, then `this.listen` can be used multiple times throughout. Makes it easier to change default values later I think.

Reply:
Ah okay sounds good will change that!

Reply:
Okay changed this to declare this.listen above, let me know if that looks better